src/sentry/incidents/models.py

```diff
@@ -41,6 +42,19 @@ class Meta:
         unique_together = (('group', 'incident'), )
 
 
+class IncidentSeen(Model):
+    __core__ = False
+
+    incident = FlexibleForeignKey('sentry.Incident')
+    user = FlexibleForeignKey(settings.AUTH_USER_MODEL, db_index=False)
```

Comment:
I'm not totally sure how safe it is to make a foreign key to `auth_user`. Any thoughts @mattrobenolt? Could be worth marking the migration as `is_dangerous` and running it manually just to be sure.

Reply:
I think it's fine, but yeah, ifo you wanna be safe, let's do it offline, then we'll have a better sense of it for the future. I don't have a better answer off the top of my head, but I _think_ it'll be fine.